code/game/objects/structures/noose.dm

```diff
@@ -0,0 +1,151 @@
+/obj/item/stack/cable_coil/building_checks(datum/stack_recipe/R, multiplier)
+	if(R.result_type == /obj/structure/chair/noose)
+		if(!(locate(/obj/structure/chair) in get_turf(usr)))
+			to_chat(usr, span_warning("You have to be standing on top of a chair to make a noose!"))
+			return FALSE
+	return ..()
+
+/obj/structure/chair/noose //It's a "chair".
+	name = "noose"
+	desc = "Hang in there!"
+	icon_state = "noose"
+	icon = 'icons/obj/noose.dmi'
+	layer = FLY_LAYER
+	flags_1 = NODECONSTRUCT_1
+	var/mutable_appearance/overlay
+
+/obj/structure/chair/noose/attackby(obj/item/W, mob/user, params)
+	if(W.tool_behaviour == TOOL_WIRECUTTER)
```

Comment:
can save a layer of indentation here by reverting the check
```suggestion
	if(W.tool_behaviour != TOOL_WIRECUTTER)
		return ..()
	[the rest of code 1 layer thinner]
```

Reply:
done